src/static/assets/css/markdown.css

```diff
@@ -0,0 +1,695 @@
+@font-face {
```

Comment:
https://github.com/sindresorhus/github-markdown-css

MIT license.  We can't just link it because I've already changed things--the link color and not setting its own font size.  I'll add a comment.